src/components/MeetingListPage/components/MeetingList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { MeetingList as MeetingListType } from 'meeting-v2';
+
+import { getMeetings } from '../../../../api/v2/meeting';
+import { getRegionId } from '../../../../util/utils';
+import MeetingCard from '../MeetingCard';
+import LineDivider from '../MeetingCard/LineDivider';
+
+function MeetingList() {
+  const [meetings, setMeetings] = useState<MeetingListType[]>([]);
+
+  const meetingListHandler = useCallback(async () => {
+    const region_id = getRegionId(window.location.search);
+    if (!region_id) return;
+    const result = await getMeetings(region_id);
+    if (result.success && result.data) setMeetings(result.data);
+  }, [setMeetings]);
+
+  useEffect(() => {
+    meetingListHandler();
+  }, [meetingListHandler]);
```

Comment:
P4;

문득 든 생각인데 meetingListHandler 부분(L11-L22)을 hooks로 만드는 것은 어떨까 하는 생각이 들었어요.
해당 컴포넌트에서는 어떤 데이터를 어떻게 render 할 지에 집중해서 보여주고, 데이터를 가져오는 과정에서 핸들링 하는 로직은 따로 분리하면 재사용성도 좋아지고 관심사도 좋게 분리되지 않을까? 하는 생각이에요.

```suggestion
  const useGetMeetingList = (regionId: string) => {
  	const [meetings, setMeetings] = useState<MeetingListType[]>([]);
  	const [fetchingState, setFetchingState] = useState<'LOADING' | 'SUCCESS' | 'ERROR'>('LOADING');
  	
  	const meetingListHandler = useCallback(async () => {
    		if (!region_id) return;
    		
    		const result = await getMeetings(region_id);
    		
    		if (result.success && result.data) setMeetings(result.data);
	}, [setMeetings]);

 	 useEffect(() => {
    		meetingListHandler();
	}, [meetingListHandler]);
	
	return { meetings, fetchingState};
  }

  //  MeetingList Component
  const { meetings } = useGetMeetingList();
```

Reply:
오오,,,, 이것도 너무 좋은데요?! ㅎㅎㅎ
확실히 훅으로 분리시키는게 깔끔하고 활용성도 높아보여요!! P1으로 반영할게요ㅋㅋㅋ 
감사해요 루트!